src/internal/context/scrollbar-label-context.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { createContext } from 'react';
+
+export const ScrollbarLabelContext = createContext<string | undefined>(undefined);
```

Comment:
Why is this called `ScrollbarLabelContext`? It sounds like we're adding a label to a scrollbar.

Reply:
Naming is difficult! We are adding a label to table / cards kinda because of a scrollbar. But maybe, `CollectionLabelContext` sounds better